packages/react-tabs/src/components/Pivot/Pivot.base.tsx

```diff
@@ -66,13 +65,25 @@ const isPivotItem = (item: React.ReactNode): item is PivotItem => {
 
 export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef<HTMLDivElement, IPivotProps>(
   (props, ref) => {
-    const { componentRef, theme, linkSize, linkFormat, overflowBehavior } = props;
-    const pivotId: string = useId('Pivot');
-    let linkCollection = getLinkItems(props, pivotId);
     const focusZoneRef = React.useRef<IFocusZone>(null);
-    const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(props, divProperties);
+    const overflowMenuButtonComponentRef = React.useRef<IButton>(null);
+    const pivotId: string = useId('Pivot');
+
     const [selectedKey, setSelectedKey] = useControllableValue(props.selectedKey, props.defaultSelectedKey);
+
+    const { componentRef, theme, linkSize, linkFormat, overflowBehavior } = props;
+    // The overflow menu starts empty and items[] is updated as the overflow items change
+    const overflowMenuProps: IContextualMenuProps = {
```

Comment:
Not new but this should probably be memoized (or even useConst) so it doesn't mutate every render. I can actually see that being a source of bugs too, with the way it's used.